arviz/__init__.py

```diff
@@ -1,6 +1,6 @@
 # pylint: disable=wildcard-import,invalid-name,wrong-import-position
 """ArviZ is a library for exploratory analysis of Bayesian models."""
-__version__ = "0.15.0"
+__version__ = "0.16.0dev"
```

Comment:
I dont know if I did this right. Should it be 0.15.1dev0? How do we know there wont be a bugfix bump?


https://python.arviz.org/en/latest/contributing/how_to_release.html

Reply:
we can still release a patch version if necessary, but in general we start merging new features as soon as something was released, so the common situation is a minor bump. Also note the syntax to make sure it is pep compiant:

```suggestion
__version__ = "0.16.0.dev0"
```